### PR TITLE
manual: fix more broken HTML tags

### PR DESCRIPTION
**Pull request by neteler**
In order to obtain a good result when converting the HTML manual pages files to markdown, ideally clean HTML code is used.

This PR fixes numerous broken HTML tags identified during a test run of [adding HTML validation with super-linter](https://github.com/echoix/grass/pull/303).

For reference, see logs showing the errors at
https://github.com/echoix/grass/actions/runs/12001379426/job/33451802353?pr=303#step:4:8310

Fixes (there is not much variety here, just many occurrences):
- complete incomplete tag pairs (esp. `<li>`)
- tabs replaced with white space
- `<`, `>`, `&` replaced with their HTML representations
- avoid `<br>` within `<em> ... </em>` since this will not be properly converted by `pandoc`
- update an URL

Testing: the simplest approach is to see if the code still compiles. Locally it does.

### CODE REVIEW

File: imagery/i.ortho.photo/i.ortho.camera/i.ortho.camera.html [hunk 1]
```diff
@@ -73,20 +73,20 @@ <h2>DESCRIPTION</h2>
...
+    3__    _____        0.0___    0.0___
+    4__    _____        0.0___    0.0___
+    5__    _____        0.0___    0.0___
+    6__    _____        0.0___    0.0___
+    7__    _____        0.0___    0.0___
+    8__    _____        0.0___    0.0___
+    9__    _____        0.0___    0.0___
+    10_    _____        0.0___    0.0___
+
+             next:  end__
```
Comment (echoix) [inline imagery/i.ortho.photo/i.ortho.camera/i.ortho.camera.html:89]: It's ok for the docs, but the tool really does output tabs?

**Review by echoix — APPROVED** [review]
I've left one comment. And also saw that an url to grassbook was changed to neteler

File: imagery/i.ortho.photo/i.ortho.camera/i.ortho.camera.html (hunk 1 above)
Comment (echoix) [reply imagery/i.ortho.photo/i.ortho.camera/i.ortho.camera.html:89]: I don't see if it's tabs or spaces. Just noticing that if the tool was outputting tabs, now the docs would have spaces.

**Comment by echoix** [comment]
For the tabs vs spaces, line endings, end of line, etc, there's an editorconfig file that can be useful, and enforced with editorconfig-checker. Editorconfig-checker can be enabled in super-linter too.  I've started the work of preparing the repo for it months ago, that's where the 2 still unmerged PRs changing shell scripts came from (very inconsistent tabs vs spaces). I think Python is ready now (or almost). Anyways, it's only another tool up the sleeve for this kind of checks.

File: imagery/i.ortho.photo/i.ortho.camera/i.ortho.camera.html (hunk 1 above)
Comment (neteler) [reply imagery/i.ortho.photo/i.ortho.camera/i.ortho.camera.html:89]: It is spaces now and looks as before:

image

**Comment by neteler** [comment]
> For the tabs vs spaces, line endings, end of line, etc, there's an editorconfig file that can be useful, and enforced with editorconfig-checker. Editorconfig-checker can be enabled in super-linter too. 

Additionally, it would be good to also update:
https://grasswiki.osgeo.org/wiki/Tools_for_Python_programming

**Review by echoix — APPROVED** [review]

**Pull request merged**